.github/workflows/cd.yaml

```diff
@@ -0,0 +1,15 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+
+name: cd
+on:
+  workflow_dispatch:
+  check_run:
+    types:
+      - completed
+
+jobs:
+  maven-cd:
+    uses: jenkins-infra/github-reusable-workflows/.github/workflows/maven-cd.yml@v1
```

Comment:
Should this be `jenkins-infra/jenkins-maven-cd-action@v1.2.0` ? I got this line from [a template here](https://raw.githubusercontent.com/jenkinsci/.github/master/workflow-templates/cd.yaml) but I have seen a few [examples](https://github.com/jenkinsci/kubernetes-plugin/pull/1139/files) where it is different.

Reply:
Just make sure to follow https://www.jenkins.io/doc/developer/publishing/releasing-cd/#configure-cd-workflow ?

Reply:
Yes, I am following that, just noted that there are differences between that and other existing PRs so wanted to check I was on the right path 🙂

Reply:
the difference is between using a re-usable workflow (the `uses` part here) and duplicating it all instead.
GitHub reusable workflows came some point after the continuous deleivery of plugins - hence some older ones are not using the reusable workflow.